### ISSUE TITLE
param name for C/unsigned types

### ISSUE THREAD

**Issue by ghost** [20Tauri/DoxyDoxygen#34]
When the C function has a unsigned type parameter, the doxydoxygen cannot identify the variable name. It puts them as `unnamed`.

For e.g.:

```
/**
 * @brief      { function_description }
 *
 * @param[in]  <unnamed>  { parameter_description }
 */
void foo(unsigned int var1);
```

**Comment by 20Tauri** [comment]
Thank you for reporting.
I have just fixed it in 0.30.5 version (evolution branch).
Will be fixed in regular version soon

**Comment by 20Tauri** [comment]
Released in both versions
